app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/apps/ui/TrackingProtectionExclusionListActivity.kt

```diff
@@ -90,6 +92,8 @@ class TrackingProtectionExclusionListActivity :
         bindViews()
         observeViewModel()
 
+        viewModel.applyAppsFilter(getAppsFilterOrDefault())
```

Comment:
`observeViewModel` launches a coroutine that calls `viewModel.getProtectedApps()`. viewModel.applyAppsFilter() will also trigger the same pipeline. I think you can remove one of the two calls.